lib/vsc/filesystem/quota/__init__.py

```diff
@@ -0,0 +1,2 @@
+import pkg_resources
+pkg_resources.declare_namespace(__name__)
```

Comment:
traditionally we have this:

```python
from pkgutil import extend_path

# we're not the only ones in this namespace
__path__ = extend_path(__path__, __name__)  #@ReservedAssignment
```

but maybe that's because we had to support Python 2.4?